pkgs/applications/networking/browsers/qutebrowser/default.nix

```diff
@@ -86,7 +83,7 @@ python3.pkgs.buildPythonApplication {
     runHook preInstall
 
     make -f misc/Makefile \
-      PYTHON=${buildPackages.python3}/bin/python3 \
+      PYTHON=${python3.pythonOnBuildForHost.interpreter} \
```

Comment:
You can also remove the `buildPackages` argument from the top of the file now as well